src/encoded/types/functional_characterization_experiment.py

```diff
@@ -205,7 +205,7 @@ def perturbation_type(self, request, assay_term_name, replicates=None):
     })
     def crispr_screen_readout(self, request, assay_term_name, examined_loci=None, control_type=None):
         crispr_screen_readout = None
-        if assay_term_name in ['proliferation CRISPR screen', 'FACS CRISPR screen', 'Flow-FISH CRISPR screen']:
+        if assay_term_name in ['proliferation CRISPR screen', 'FACS CRISPR screen', 'Flow-FISH CRISPR screen', 'CRISPR screen']:
             # Return a specific CRISPR screen readout if there is only one method used in examined_loci, no examined_loci at all indicates a growth screen
             if control_type == 'control':
```

Comment:
This looks like it would need to be removed in order to calculate readout on controls

Reply:
would it make sense to change this to be if "CRISPR Screen" in assay_slims?  Simpler code, but less specific in what you are doing.